Art/ParzivalAFK/HalfSpinPoints.css

```diff
@@ -0,0 +1,83 @@
+body{
+    margin: 0;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #000;
+}
+
+.container{
+    position: relative;
+}
+
+.rotateContainer{
+    position: absolute;
+}
+.ball{
+    position:absolute;
+    width: 50px;
+    height: 50px;
+    background-color: rgb(255, 248, 211);
+    animation: ball_Animation 8s linear infinite alternate;
+    border-radius: 100%;
+}
+
+@keyframes ball_Animation {
+    0%{
+        transform: translateX(-300px);
+    }
+    100%{
+        transform: translateX(300px);
+    }    
+}
+
+.rotate_1{
+    transform: rotate(0deg);
+}
+.rotate_2{
+    transform: rotate(22.5deg);
+}
+.rotate_3{
+    transform: rotate(45deg);
+}
+.rotate_4{
+    transform: rotate(67.5deg);
+}
+.rotate_5{
+    transform: rotate(90deg);
+}
+.rotate_6{
+    transform: rotate(112.5deg);
+}
+.rotate_7{
+    transform: rotate(135deg);
+}
+.rotate_8{
+    transform: rotate(157.5deg);
+}
+/* 
+.ball_1{
+    animation-delay: 0s;
+}
+.ball_2{
+    animation-delay: 1s;
+}
+.ball_3{
+    animation-delay: 2s;
+}
+.ball_4{
+    animation-delay: 3s;
+}
+.ball_5{
+    animation-delay: 4s;
+}
+.ball_6{
+    animation-delay: 5s;
+}
+.ball_7{
+    animation-delay: 6s;
+}
+.ball_8{
+    animation-delay: 7s;
+} */
```

Comment:
This is a duplicate of a previous pull request, if I remember correctly.

Please sync your fork with the master branch, see [https://short.pm/forksync](https://short.pm/forksync).